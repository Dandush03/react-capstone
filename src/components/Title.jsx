import React from 'react';

export default function Title() {
  return (
    <div className="title">
      <h1>Assets Comparations</h1>
    </div>
  );
}
