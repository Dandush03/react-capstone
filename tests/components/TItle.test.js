import React from 'react';
import renderer from 'react-test-renderer';
import Title from '../../src/components/Title';

describe('Title', () => {
  test('Render Elements', () => {
    const component = renderer.create(<Title />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
