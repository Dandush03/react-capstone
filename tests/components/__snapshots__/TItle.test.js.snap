// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title Render Elements 1`] = `
<div
  className="title"
>
  <h1>
    Assets Comparations
  </h1>
</div>
`;
